public/docs/_examples/dependency-injection/dart/lib/car/car_factory.dart

```diff
@@ -1,7 +1,7 @@
 // #docregion
-
 import 'car.dart';
 
+// BAD pattern!
 class CarFactory {
   Car createCar() {
     var car = new Car(createEngine(), createTires());
```

Comment:
Could showcase the cascade operator here:

return new Car(createEngine(), createTires())..description = 'Factory';